api/src/db/models/legacy_mixin/certificate_mixin.py

```diff
@@ -0,0 +1,32 @@
+#
+# SQLAlchemy models for foreign tables.
+#
+# The order of the columns must match the remote Oracle database. The names are not required to
+# match by oracle_fdw, but we are matching them for maintainability.
+#
+
+from datetime import date, datetime
+
+from sqlalchemy.orm import Mapped, declarative_mixin, mapped_column
+
+
+@declarative_mixin
+class TcertificateMixin:
+    currentcertid: Mapped[str] = mapped_column(
+        primary_key=True
+    )  # currentcertid - text, primary key
+    previouscertid: Mapped[str | None]  # previouscertid - text
```

Comment:
Could we trim out the comments? The table itself says the same thing albeit in the python types.